### PR TITLE
Update build.md

### PR DESCRIPTION
**Pull request by Lodewic**
Added instructions how to install the GPU-enabled R package for Windows. As requested by @thirdwing https://github.com/dmlc/mxnet/issues/1085

### CODE REVIEW

File: doc/build.md [hunk 1]
```diff
@@ -239,6 +239,33 @@ Now you should have the R package as a tar.gz file and you can install it as a n
...
 ```
 
+
+To install the package using GPU on Windows without building the package from scratch. Note that you need a couple of programs installed already:  
+- You'll need the [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit). This depends on Visual Studio, and a free compatible version would be [Visual Studio Community 2013](https://www.visualstudio.com/en-us/news/vs2013-community-vs.aspx). For instructions and compatibility checks, read http://docs.nvidia.com/cuda/cuda-getting-started-guide-for-microsoft-windows/ .
+
+- You will also need to register as a developer at nvidia and download CUDNN V3, https://developer.nvidia.com/cudnn . 
+
+
+(1) Download the mxnet package as a ZIP from the Github repository https://github.com/dmlc/mxnet and unpack it. You will be editing the `/mxnet/R-package` folder.
```
Comment (thirdwing) [inline doc/build.md]: Just to keep consistent with other parts, please change `(1)` into `1.` Others similar.

File: doc/build.md [hunk 2]
```diff
@@ -239,6 +239,33 @@ Now you should have the R package as a tar.gz file and you can install it as a n
...
+
+(5) put dll files in `R-package/inst/libs/x64`. 
+
+The first dll file you need is `nocudnn/lib/libmxnet.dll`. The other dll files you need are the ones in all 4 subfolders of `nocudnn/3rdparty/`, for the `cudnn` and `openblas` you'll need to look in the `/bin` folders. There should be 11 dll files now in `R-package/inst/libs/x64`.
+
+(6) copy the folder `nocudnn/include/` to `R-package/inst/`. So now you should have a folder `R-package/inst/include/` with 3 subfolders.
+
+(7) `R CMD INSTALL R-package`. Make sure that R is added to your PATH in Environment Variables. Running the command `Where R` in Command Prompt should return the location.
+
+You might get an error similar to `Error: loading failed for 'i386'`. In that case, run `R CMD INSTALL --no-multiarch R-package`.
```
Comment (thirdwing) [inline doc/build.md]: We always need `R CMD INSTALL --no-multiarch R-package`, so maybe you need change the last step a little.

**Comment by thirdwing** [comment]
Hi, you just need to make some very minor changes, then I will merge it.

**Comment by tqchen** [comment]
Thanks for this contribution! have added you to https://github.com/dmlc/mxnet/blob/master/CONTRIBUTORS.md

Please feel free to push more changes, and help making mxnet and mxnetR more accessible for the community

**Pull request merged**